python/python/lance/ray/sink.py

```diff
@@ -213,24 +219,43 @@ class LanceFragmentWriter:
     in distributed fashion.
 
     Parameters
+    ----------
+    uri : str
+        The base URI of the dataset.
+    transform : Callable[[pa.Table], Union[pa.Table, Generator]], optional
+        A callable to transform the input batch. Default is identity.
+    schema : pyarrow.Schema, optional
+        The schema of the dataset.
+    max_rows_per_file : int, optional
+        The maximum number of rows per file. Default is 1024 * 1024.
+    max_bytes_per_file : int, optional
+        The maximum number of bytes per file. Default is None.
+    max_rows_per_group : int, optional
+        The maximum number of rows per group. Default is 1024.
+        Only useful for v1 writer.
+    use_experimental_writer : bool, optional
+        Set true to use v2 writer. Default is True.
+
     """
 
     def __init__(
         self,
         uri: str,
         *,
-        transform: Callable[[pa.Table], Union[pa.Table, Generator]] = lambda x: x,
+        transform: Callable[[pa.Table], Union[pa.Table, Generator]] = None,
```

Comment:
If you're going to make the default `None`, then you have to change the type accordingly:
```suggestion
        transform: Optional[Callable[[pa.Table], Union[pa.Table, Generator]]] = None,
```

Reply:
Fixed.